common/test/acceptance/tests/lms/test_teams.py

```diff
@@ -455,3 +456,192 @@ def test_teams_membership(self):
             self.browse_teams_page.team_cards[0].find_element_by_css_selector('.member-count').text,
             '1 / 10 Members'
         )
+
+    def test_navigation_links_are_working_properly(self):
```

Comment:
Nit: remove "_are_working_properly" from the name of this method, since all of our test methods are supposed to verify that things are working properly :)